media/CircleEditor/view-sidebar.js

```diff
@@ -993,9 +1016,24 @@ sidebar.ParameterView = class {
 };
 
 sidebar.ArgumentView = class {
-    constructor(host, argument) {
+    constructor(host, argument, tensors, ioAttributes) {
         this._host = host;
         this._argument = argument;
+        this._select;
+        this._shape;
+        this._data;
+        this._index = ioAttributes.index;
+        this._isCustom = ioAttributes.isCustom;
+        this._tensors = tensors;
+        this._title = ioAttributes.title;
+
+        this._editObject = {
+            _name: ioAttributes.name,
+            _visible: ioAttributes.visible,
+            _arguments: JSON.parse(JSON.stringify(argument)),
```

Comment:
Isn't it `JSON.parse(JSON.stringify(argument)) === argument` ?

Reply:
If do not use this method when sending an object value, the value itself may change even if the value is not valid.  So we have to deep copy.